packages/client/src/app/services/network.service.ts

```diff
@@ -245,7 +245,7 @@ export class NetworkService {
                 catchError((err) => {
                     if (err.status === 403) {
                         // A 403 means that the refreshToken has expired, or we didn't send one up at all, which is Super Suspicious
-                        return null;
+                        return of<string>("expired");
```

Comment:
I think you can actually do `of<string>(null)` here. That way the return signature of the method remains true, and you can keep the null check down below instead of checking for a magic string. My mistake here was that this was supposed to return an `Observable<T>`, and not a raw value. (I thought it'd make the already-return observable emit whatever I returned. It does not!)